src/EditorFeatures/CSharp/CompleteStatement/CompleteStatementCommandHandler.cs

```diff
@@ -317,87 +317,6 @@ private static bool IsInAStringOrCharacter(SyntaxNode currentNode, SnapshotPoint
                 && caret.Position < currentNode.Span.End
                 && caret.Position > currentNode.SpanStart;
 
-        private static bool SemicolonIsMissing(SyntaxNode currentNode)
```

Comment:
@sharwell i'm not sure how this command handler is supposed to work.  can you take a look?  Seems veryfishythat this isn't being used.

Reply:
@CyrusNajmabadi I'm unsure too, that's why I opened as draft to see if the CI will fail.
If this is being used, I'd like @sharwell to confirm whether it should be updated with records (it currently have a check for class)